packages/datadog-plugin-express/test/integration-test/client.spec.js

```diff
@@ -36,6 +36,7 @@ describe('esm', () => {
     })
 
     it('is instrumented', async () => {
+      delete process.env.DD_TRACE_MIDDLEWARE_ENABLED
```

Comment:
Altering environment variables should be done in `beforeEach` and `afterEach` to make sure that the previous value is restored.